src/tests/ftest/config_file_gen.py

```diff
@@ -51,6 +51,12 @@ def generate_server_config(args):
     config.engine_params[0].storage.storage_tiers[0].storage_class.value = "ram"
     config.engine_params[0].storage.storage_tiers[0].scm_mount.value = "/mnt/daos"
     config.engine_params[0].storage.storage_tiers[0].scm_size.value = 0
+
+    # Disable shutdown state (SDS) (part of RAS) for RAM-based simulated SCM.
+    # RAM doesn't support this feature and trying to use it will fail the create/open operations.
+    # PMEMOBJ expects the shutdown state support by default when built with NDCTL.
+    config.engine_params[0].env_vars.update("PMEMOBJ_CONF=sds.at_create=0", append=True)
```

Comment:
Is this an env var that everyone who uses DAOS with ramdisk will need? If so, we need to consider populating it automatically, too.

Reply:
Done. Every time both PMDK-related env variables are generated before engine is started.